packages/core/src/preview.ts

```diff
@@ -79,19 +86,29 @@ export const getPreviewString = (
         const element = executing[i]
 
         if (isDecodedAction(element)) {
-          const { referenceName, functionName, variables, address } = element
+          const { referenceName, functionName, variables, address, value } =
+            element
           const actionStr = prettyFunctionCall(
             referenceName,
             address,
             functionName,
             variables,
+            value,
             5,
             3
           )
 
           executingArray.push(green(`${i + 1}. ${actionStr}`))
         } else if (isSystemDeploymentElement(element)) {
           executingArray.push(green(`${i + 1}. Sphinx & Gnosis Safe Contracts`))
+        } else if (isFundingSafeElement(element)) {
+          executingArray.push(
+            green(
+              `${i + 1}. Transfer funds to Gnosis Safe: ${formatEther(
```

Comment:
<img width="852" alt="Screen Shot 2024-03-19 at 3 04 53 PM" src="https://github.com/sphinx-labs/sphinx/assets/106038229/16f225db-b142-41d4-87cd-177cad03c588">

What are your thoughts on keeping the syntax consistent with the other preview elements and doing something like:
```
GnosisSafe<0x...>.call{ value: ... }();
```

I'm not opinionated on the syntax, but I think we should at least include the Gnosis Safe's address because otherwise it won't be displayed in the preview at all (if the Gnosis Safe has already been deployed)